RELEASE.md

```diff
@@ -12,6 +14,11 @@
 * Added style sheets for project documentation, so the output of `kedro build-docs` will resemble the style of `kedro docs`.
 
 ## Breaking changes to the API
+* `src/requirements.yml` will be required to run `kedro install --conda`
```

Comment:
This has to be added to the existing "Breaking changes to the API" section for 0.15. Probably the result of merge conflict. Please have a look at the current state of `RELEASE.md` on `develop`.